<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>login</class>
 <widget class="QDialog" name="login" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>231</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Login</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="..\hall.qrc" >:/HallRes/image/login.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="labelName" >
       <property name="text" >
        <string>user name:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="buddy" >
        <cstring>comboName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboName" >
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::InsertAtBottom</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="labelPassword" >
       <property name="text" >
        <string>password:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="buddy" >
        <cstring>lePassword</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePassword" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbSaveName" >
       <property name="text" >
        <string>save user name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSavePassword" >
       <property name="text" >
        <string>save password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnRegister" >
       <property name="text" >
        <string>Register</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogin" >
       <property name="text" >
        <string>Login</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>comboName</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>cbSaveName</tabstop>
  <tabstop>cbSavePassword</tabstop>
  <tabstop>btnRegister</tabstop>
  <tabstop>btnLogin</tabstop>
 </tabstops>
 <resources>
  <include location="..\hall.qrc" />
 </resources>
 <connections/>
</ui>
